app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -27,6 +27,11 @@ body {
   padding: 0px;
 }
 
+html, body, #app {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line